src/bundles/robot_simulation/controllers/ev3/ev3/default/config.ts

```diff
@@ -0,0 +1,155 @@
+import type {
+  Ev3ChassisConfig,
+  Ev3ColorSensorConfig,
+  Ev3Config,
+  Ev3MeshConfig,
+  Ev3MotorsConfig,
+  Ev3UltrasonicSenorConfig,
+  Ev3WheelsConfig,
+} from './types';
+
+const noRotation = {
+  x: 0,
+  y: 0,
+  z: 0,
+  w: 1,
+};
+
+const tinyConstant = 0.012;
+
+export const chassisConfig: Ev3ChassisConfig = {
+  orientation: {
+    position: {
+      x: 0,
+      y: 0.0775,
+      z: 0,
+    },
+    rotation: noRotation,
+  },
+  dimension: {
+    height: 0.095,
+    width: 0.145,
+    length: 0.18,
+  },
+  mass: 0.6,
+  type: 'dynamic',
+  debug: true,
+};
+
+export const meshConfig: Ev3MeshConfig = {
+  url: 'https://keen-longma-3c1be1.netlify.app/6_remove_wheels.gltf',
```

Comment:
Are we going to eventually move this into a/the repo?

Reply:
Hello! Sorry for the slow reply! 

How do other modules host their static files? Is it inside the modules repo or in a separate repo?

Reply:
So far I think the only other module with static assets is unity academy, and we host the files in an S3 bucket on AWS

Reply:
Ah alright. How do I go about to do that?

Reply:
Let's take this over to Telegram.